Sources/PreviewGallery/PreviewData.swift

```diff
@@ -8,22 +8,45 @@
 import Foundation
 import SnapshotPreviewsCore
 
+
+/// A structure that manages a collection of preview types.
+///
+/// `PreviewData` is used to store and manipulate a set of `PreviewType` instances,
```

Comment:
I think this could probably be re-written, something like "The backing data for PreviewGallery" might be all it needs. It doesn't really have any publicly accessible functionality